client/src/panels/panel_declarations/finances/drr_dp_resources/sub_program_resources.js

```diff
@@ -154,6 +152,54 @@ const get_non_col_renderer = ({doc}) => ({node}) => {
   );
 };
 
+const get_col_defs = ({doc}) => [
```

Comment:
Commented on this in the relevant commit message, but this is a copy of the old `get_col_defs` from resource_explorer_common.js. I'm fine with a dumb copy paste of it here because this module is slated for deletion soon enough anyway,

Reply:
:+1: